src/styles/blocks/general.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
Move font styles to the html selector

Reply:
why i need this fixed> realle? in goit we alwaye resets with * and this will ok? wtats problem with this?

Reply:
using the universal selector (*) in CSS can have significant performance and specificity implications, especially if used for more than just the box-sizing property
<img width="609" alt="image" src="https://github.com/user-attachments/assets/60a0a0bf-8b87-4569-9bf6-1404c75caaf0">


feel free to ask for more details in the chat